packages/firestore/src/platform_browser/webchannel_connection.ts

```diff
@@ -310,7 +308,7 @@ export class WebChannelConnection implements Connection {
     // Note that eventually this function could go away if we are confident
     // enough the code is exception free.
     const unguardedEventListen = <T>(
-      type: WebChannel.EventType,
+      type: string,
```

Comment:
Was it not possible to make WebChannel.EventType work?  It seems like we're losing a bit of help from the type system here by switching to just `string`.  Not the end of the world though.

Reply:
WebChannel.EventType is now a variable declaration and no longer a string. I had tried both `WebChannel.EventType` and `typepof WebChannel.EventType`.

Reply:
Bleh...  I came up with several ways to get this to compile (let us use `EventType` as a type as well as hang property values off it).  E.g.:

```
export interface EventType {}
export var EventType: {
  readonly COMPLETE: EventType
}
```
or:
```
export class EventType {
  static readonly COMPLETE: EventType;
}
```

but they didn't provide type-safety, since TypeScript's structural typing would let you pass anything (e.g. `{x: 1}`) for a parameter typed as `EventType`. 😕 

So the only thing I found that actually works for type-safety is using `export enum EventType { COMPLETE }` like you originally had.  So if you feel like putting that back, I would be okay with that.  Else if you're tired of messing with it, this is okay too.

Reply:
Hm. My "I couldn't get it to work" response above is based on the current version that used the variable declaration w/ nested property syntax. I am inclined to leave that as is for now, as it matches the underlying library closer than the Enum values.